fe/fe-core/src/main/java/com/starrocks/external/starrocks/TableMetaSyncer.java

```diff
@@ -34,7 +34,7 @@ public void syncTable(ExternalOlapTable table) {
 
         TGetTableMetaRequest request = new TGetTableMetaRequest();
         request.setDb_name(table.getExternalInfo().getDbName());
-        request.setTable_name(table.getName());
+        request.setTable_name(table.getExternalInfo().getTableName());
```

Comment:
Better to use db id and table id, which can be compatible with remote db and table renaming.

Reply:
For the first time get table meta from remote cluster, we have nor idea of table id